tests/test_transforms.py

```diff
@@ -310,6 +310,7 @@ def dynamic_chunking_fn(template_ds: xr.Dataset, divisor: int = 1):
             combine_dims=pattern.combine_dim_keys,
             attrs={},
             dynamic_chunking_fn=dynamic_chunking_fn,
+            consolidate_dimension_coordinates=False,
             consolidated_metadata=True,
```

Comment:
```suggestion
```

Assuming we remove these flags as proposed above, they would need to be removed from this test as well.